api/endpoints_debug.go

```diff
@@ -65,7 +65,7 @@ func printStack(_ *Request) (msg string, err error) {
 // debugInfo returns the debugging information for support requests.
 func debugInfo(ar *Request) (data []byte, err error) {
 	// Create debug information helper.
-	di := new(osdetail.DebugInfo)
+	di := new(debug.DebugInfo)
 	di.Style = ar.Request.URL.Query().Get("style")
```

Comment:
I'd rename `debug.DebugInfo` to  `debug.Info` (go-vet should complain about it)

Reply:
I figured, but wasn't up for it at the moment.

Reply:
Will fix ;)